### PR TITLE
add beta2 suffix for dev16 branch

### PR DESCRIPTION
**Pull request by brettfo**

### CODE REVIEW

File: build/config/PublishData.json [hunk 1]
```diff
@@ -28,6 +28,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev15.9p1"]
         },
+        "dev16.0.x-vs-deps": {
```
Comment (Pilchie) [inline build/config/PublishData.json]: I suspect this part of the change needs to happen in the `dev16.0.x-vs-deps` branch?

File: build/config/PublishData.json [hunk 2]
```diff
@@ -28,6 +28,13 @@
...
+            "nugetKind": "PerBuildPreRelease",
+            "version": "2.11.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.0", "dev16.0p1" ]
```
Comment (jaredpar) [inline build/config/PublishData.json]: Lets change the channels entry for master-vs-deps to be dev16.0p2 as well here.

File: build/config/PublishData.json (hunk 1 above)
Comment (jasonmalinowski) [reply build/config/PublishData.json]: @Pilchie: no, the publishing tools always reads it out of the master branch.

File: build/config/PublishData.json (hunk 2 above)
Comment (jasonmalinowski) [reply build/config/PublishData.json]: Actually I believe this channel should be _just_ 16.0p1 and dev16.0 goes on master-vs-deps too?

Comment (jaredpar) [reply build/config/PublishData.json]: Yes that's correct.

**Comment by brettfo** [comment]
Updated this PR to only be the `-beta2` suffix.

I created #30730 to track the package publishing out of master.

**Comment by jasonmalinowski** [comment]
Doh, this wasn't supposed to be targeting dev16.0.x -- I've already undone this in 2ab637fee75c9f8aedf24b503f954e895e477bf2.

**Pull request merged**